# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/frame_builder.rs
# Commit: a0b3e2669a2d
# Full Hash: a0b3e2669a2dc49c467521cb9b7966ca05e3dbc3
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-05-08 09:49:23
# Regressor Bug: 1824502
# File Overlap Count: 1
# Description:
#   Bug 1824502 - Implement native SVG filter rendering in WebRender r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209646
# ==============================================================================

diff -r d1df40ba7c88 -r a0b3e2669a2d gfx/wr/webrender/src/frame_builder.rs
--- a/gfx/wr/webrender/src/frame_builder.rs	Wed May 08 01:43:25 2024 +0000
+++ b/gfx/wr/webrender/src/frame_builder.rs	Wed May 08 04:38:26 2024 +0000
@@ -453,6 +453,7 @@
                     SubpixelMode::Allow,
                     &mut frame_state,
                     &frame_context,
+                    data_stores,
                     &mut scratch.primitive,
                     tile_caches,
                 )